src/components/ClusterNodes/ClusterList.tsx

```diff
@@ -0,0 +1,204 @@
+import React, { useState, useEffect } from 'react'
+import { NavLink } from 'react-router-dom'
+import { useRouteMatch } from 'react-router'
+import './clusterNodes.scss'
+import { ReactComponent as Search } from '../../assets/icons/ic-search.svg'
+import { ReactComponent as Clear } from '../../assets/icons/ic-error.svg'
+import { getClusterList } from './clusterNodes.service'
+import { handleUTCTime, Progressing, showError } from '../common'
+import { ClusterDetail, ClusterListResponse } from './types'
+import PageHeader from '../common/header/PageHeader'
+import { toast } from 'react-toastify'
+import { ReactComponent as Error } from '../../assets/icons/ic-error-exclamation.svg'
+import { ReactComponent as Success } from '../../assets/icons/appstatus/healthy.svg'
+import ClusterNodeEmptyState from './ClusterNodeEmptyStates'
+import Tippy from '@tippyjs/react'
+
+export default function ClusterList() {
+    const match = useRouteMatch()
+    const [loader, setLoader] = useState(false)
+    const [noResults, setNoResults] = useState(false)
+    const [searchText, setSearchText] = useState('')
+    const [filteredClusterList, setFilteredClusterList] = useState<ClusterDetail[]>([])
+    const [clusterList, setClusterList] = useState<ClusterDetail[]>([])
+    const [lastDataSyncTimeString, setLastDataSyncTimeString] = useState('')
+    const [lastDataSync, setLastDataSync] = useState(false)
+    const [searchApplied, setSearchApplied] = useState(false)
+
+    const getData = () => {
+        setLoader(true)
+        getClusterList()
+            .then((response: ClusterListResponse) => {
+                setLastDataSync(!lastDataSync)
+                if (response.result) {
+                    const sortedResult = response.result.sort((a, b) => a['name'].localeCompare(b['name']))
+                    setClusterList(sortedResult)
+                    setFilteredClusterList(sortedResult)
+                }
+                setLoader(false)
+            })
+            .catch((error) => {
+                showError(error)
+                setLoader(false)
+            })
+    }
+
+    useEffect(() => {
+        getData()
+    }, [])
+
+    useEffect(() => {
+        const _lastDataSyncTime = Date()
+        setLastDataSyncTimeString('Last refreshed ' + handleUTCTime(_lastDataSyncTime, true))
+        const interval = setInterval(() => {
+            setLastDataSyncTimeString('Last refreshed ' + handleUTCTime(_lastDataSyncTime, true))
+        }, 1000)
```

Comment:
Do we want to update the last sync time every 1s?

Reply:
NVM got the idea :)

Reply:
Pick this logic from Applist. though we can change it to 30/60 sec as well but that can show incorrect time for some seconds

Reply:
Got it